ibexa/commerce/4.6/config/packages/ibexa_twig_components.yaml

```diff
@@ -0,0 +1,24 @@
+## Uncomment configuration below if you wish to configure twig components
+
+#ibexa_twig_components:
```

Comment:
Not very keen on commented config that needs to be adjusted - I think thos should be either part of doc, or placed inside configuration class in bundle